### DISCUSSION TITLE
記事投稿キャンペーンで記事を書いていないのに当選してしまいました

### DISCUSSION THREAD

**Discussion by M-Y-Y** [increments/qiita-discussions#657] [ヘルプ] [answered]
先日終了した「キャリアを振り返ろう」キャンペーンで、参加だけして結局記事を書かずに終わってしまったのですが先程お守りの当選メールが来ていました💦
記事を書いていないのに景品をいただくのは申し訳ないので発送先連絡はせず辞退する予定ですが、
当選条件に「記事投稿いただいた方」とあるので条件抽出が誤っている可能性がありそうです。
（もしくは実は巧妙なスパムメールか…）
ご確認をよろしくお願いします。

**Comment by tanaka-asami415** [comment, marked as answer]
@M-Y-Y 
ご報告いただきありがとうございます。

この度は混乱させてしまい大変申し訳ございません。
確認したところ、参加方法の記載について誤りがございました。正しくは「参加登録」が対象となるキャンペーンとなっておりましたので、M-Y-Yさんも対象となります。キャンペーンページも修正いたしました。
ご辞退も可能ですが、受け取っていただけると幸いです🙇

**Reply by M-Y-Y** [reply]
確認ありがとうございました。
参加しただけで当選してしまったのは恐縮ですが幸運に預からせていただきます🙇‍♀
